packages/hardhat-core/recommended-gitignore.txt

```diff
@@ -1,11 +1,14 @@
 node_modules
 .env
-coverage
-coverage.json
-typechain
-typechain-types
 
 # Hardhat files
-cache
-artifacts
+/cache
```

Comment:
this would only ignore the root `cache` and `artifacts` folders, right? what about the ones inside `test/fixture-projects`?

Reply:
This PR is about changing the `.gitignore` generated by Hardhat's project initialization, not our own `.gitignore` (which is where fixture projects are relevant)